src/cljs/rems/administration/blacklist.cljs

```diff
@@ -27,9 +27,9 @@
 
 (defn- format-rows [rows]
   (doall
-   (for [{:keys [resource user]} rows]
+   (for [{:keys [blacklist/resource blacklist/user]} rows]
```

Comment:
I wonder if this affects the greppability of namespaced keywords because destructuring doesn't give them either `:` or the match on the next rows. Perhaps using `(:blacklist/resource row)` would work out better?

Reply:
good point